packages/eslint-config-airbnb/rules/es6.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     // require parens in arrow function arguments
     'arrow-parens': 0,
     // require space before/after arrow function's arrow
-    'arrow-spacing': 0,
+    'arrow-spacing': 2,
```

Comment:
Thanks! Please also include the full options object, even though it's the default: `{ "before": true, "after": true }`
Also, please provide the link to https://github.com/eslint/eslint/blob/master/docs/rules/arrow-spacing.md in a comment above the rule.